src/main/resources/additional_properties.json

```diff
@@ -18,5 +18,266 @@
+          "type" : "object",
+          "required" : [ "insert_field" ],
+          "additionalProperties" : false,
+          "properties" : {
+            "insert_field" : {
+              "additionalProperties" : false,
+              "required" : [ "apply_to", "field", "value" ],
+              "properties" : {
+                "apply_to" : {
+                  "$ref" : "#/$defs/apply_to"
+                },
+                "field" : {
+                  "type" : "string"
+                },
+                "value" : {
+                  "type" : "string"
+                }
+              }
+            }
+          }
+        },
+        {
+          "type" : "object",
+          "required" : [ "extract_field" ],
+          "additionalProperties" : false,
+          "properties" : {
+            "extract_field" : {
+              "additionalProperties" : false,
+              "required" : [ "apply_to", "field" ],
+              "properties" : {
+                "apply_to" : {
+                  "$ref" : "#/$defs/apply_to"
+                },
+                "field" : {
+                  "type" : "string"
+                }
+              }
+            }
+          }
+        },
+        {
+          "type" : "object",
+          "required" : [ "mask_field" ],
+          "additionalProperties" : false,
+          "properties" : {
+            "mask_field" : {
+              "additionalProperties" : false,
+              "required" : [ "apply_to", "field" ],
+              "properties" : {
+                "apply_to" : {
+                  "$ref" : "#/$defs/apply_to"
+                },
+                "field" : {
+                  "type" : "array",
+                  "items" : {
+                    "type" : "string"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "type" : "object",
+          "required" : [ "replace_field" ],
+          "additionalProperties" : false,
+          "properties" : {
+            "replace_field" : {
+              "additionalProperties" : false,
+              "required" : [ "apply_to", "whitelist", "blacklist", "renames" ],
+              "properties" : {
+                "apply_to" : {
+                  "$ref" : "#/$defs/apply_to"
+                },
+                "whitelist" : {
+                  "type" : "array",
+                  "items" : {
+                    "type" : "string"
+                  }
+                },
+                "blacklist" : {
+                  "type" : "array",
+                  "items" : {
+                    "type" : "string"
+                  }
+                },
+                "renames" : {
+                  "type" : "array",
+                  "items" : {
+                    "type" : "string"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "type" : "object",
+          "required" : [ "set_schema_metadata" ],
```

Comment:
I am not sure if this should be made available on a managed service. It might allow to use invalid schema ids and versions.